app/src/main/java/com/humberto/tasky/core/presentation/designsystem/components/TaskyRadioButton.kt

```diff
@@ -0,0 +1,64 @@
+package com.humberto.tasky.core.presentation.designsystem.components
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Check
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.humberto.tasky.core.presentation.designsystem.TaskyTheme
+
+@Composable
+fun TaskyRadioButton(
+    selected: Boolean,
+    enabled: Boolean = true,
+    onClick: () -> Unit,
+    radioButtonColor: Color,
+    modifier: Modifier = Modifier
```

Comment:
When creating composables, keep the following in mind:

1. `Modifier` should be the first _optional_ parameter in your parameter list. 
2. The last parameter should not be a non-Compose lambda.
3. If a Composable is only used in this file, mark it as `private`.

Not following **no 2**, you could write the following, which would be misleading as someone would expect this lambda block to be for more Compose code:

```
@Composable
fun MyCustomComposable(
  modifier: Modifier = Modifier,
  onClick: () -> Unit
)

// Usage
MyCustomComposable(
  modifier = Modifier.size(12.dp),
) {
  // WRONG - handle onClick here
}
```

Reply:
I'll keep that in mind, thanks Kai!